tests/utils/test_merge_utils.py

```diff
@@ -0,0 +1,285 @@
+import pytest
+import jsonlines
+import boto3
+from moto import mock_s3
+
+import slapp.utils.merge_utils as mu
+
+
+@pytest.mark.parametrize(
+        "labels, expected",
+        [
+            # not 3 annotations
+            ([0, 0, 0, 1, 1], None),
```

Comment:
Less than 3?

Reply:
outdated, using the post-annotation compute_majority now